template/extra_data.go

```diff
@@ -313,10 +312,10 @@ func toQuotedTFVars(v interface{}) (string, error) {
 	return result[1 : len(result)-1], err
 }
 
-func toXML(v interface{}) (string, error) {
-	output, err := xml.Marshal(v)
-	return string(output), err
-}
+// func toXML(v interface{}) (string, error) {
+// 	output, err := xml.Marshal(v)
+// 	return string(output), err
+// }
```

Comment:
should we keep these comments?

Reply:
I kept them because we may restore it once some problem are resolved.